### PR TITLE
Fix reproducibility with torch impl

### PR DESCRIPTION
**Pull request by justjhong**
Match pytorch's batch norm momentum, embedding initialization, and default learning rate.

It seems to have been primarily the learning rate, and the momentum.

fixedjaxmrviu

### CODE REVIEW

**Comment by justjhong** [comment]
changes added to https://github.com/YosefLab/scvi-v2/pull/8
